apiary.apib

```diff
@@ -3250,6 +3250,357 @@ Update the current timer. Only possible if there is a timer running.
 
 + Response 204
 
+# Group Work Orders
+
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
+
+Add a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + performed_at: `2017-05-26T10:01:49+00:00` (string, required)
+        + description: `Pipework intervention` (string, optional)
+        + department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional)
+        + layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional)
+        + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
+        + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
+        + intervention_address (object, optional)
+            + Include Address
+            + customer (object, optional) - Customer this address refers to.
+                + type (enum[string], required)
+                    + Members
+                        + contact
+                        + company
+                + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + distance_travelled (object, optional)
```

Comment:
Work orders are for jobs to be executed in the future right, should our properties also reflect that?

Reply:
WO are made on the job, no?
So this seems fine in that case.

Reply:
Looking at some online work order templates I find things like "expected start date", "starting date"

Reply:
From what I understand about work orders is that they are made up front and assigned to someone. And after the work is done, it is completed with more details. From that perspective it's weird to have `distance_travelled` when the job is not completed yet.

Reply:
Just checked with Ghislaine and a work order can be created up front, and you can optionally specify the distance you will travel to the location. And once you've done the job you can update that number with the amount you actually travelled.

Reply:
This is renamed to `location`.